src/scss/yellow/components/_breadcrumb.scss

```diff
@@ -12,11 +12,12 @@
 
     &-item {
       &-separator {
+        @extend %typography__size-caption;
         letter-spacing: 2px;
       }
 
       &:last-child {
-        font-weight: get-weight(bold);
+        @extend %typography__color--content-extra-dark;
```

Comment:
I don't know if we need to follow BEM to name the mixin names, what do you think?

Codeclimate is even complaining about these names.

Reply:
If we decide to change names we will probably have to duplicate the file where these placeholders are declared JUST to change the names but keeping the same styles.
https://github.com/getninjas/gaiden/blob/AQ-902_new-id_category/src/scss/yellow/generic/_typography.scss

Reply:
I didn't understand very well here.

We can talk in person can be easier 😬